tests/integration/test_application.py

```diff
@@ -111,6 +113,17 @@ def test_project_managed(capsys, monkeypatch, tmp_path, project, app):
     assert captured.out == (VALID_PROJECTS_DIR / project / "stdout").read_text()
 
 
+def test_version(capsys, monkeypatch, app):
+    monkeypatch.setenv("CRAFT_DEBUG", "1")
+    monkeypatch.setenv("CRAFT_MANAGED_MODE", "1")
```

Comment:
We shouldn't need this set, right? (Realistically it shouldn't matter either way, but I'd prefer not to fake a managed environment if we don't need it for other commands)

Reply:
Probably copy-and-paste leftovers. Will remove.